### PR TITLE
Update the .gitignore file

### LINKED ISSUE #334: Update the .gitignore file
We should increase the list in .gitignore to ignore the files like `.DS_Store`. For example added in #304 by mistake.

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #334

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,11 @@ node_modules/
 rvm-installer
 rvm-installer.asc
 *.log
+
+.DS_Store
+**/.DS_Store
+/.DS_Store
+__MACOSX/
+
+yarn.lock
```
Comment (muffinresearch) [inline .gitignore]: yarn.lock should not be ignored.

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,11 @@ node_modules/
 rvm-installer
 rvm-installer.asc
 *.log
+
+.DS_Store
+**/.DS_Store
```
Comment (muffinresearch) [inline .gitignore]: Won't the previous line be enough?

File: .gitignore [hunk 3]
```diff
@@ -6,3 +6,11 @@ node_modules/
...
+/.DS_Store
```
Comment (muffinresearch) [inline .gitignore]: I think this one is unnecessary too.

File: .gitignore (hunk 1 above)
Comment (championshuttler) [reply .gitignore]: Oops my bad.

File: .gitignore (hunk 2 above)
Comment (championshuttler) [reply .gitignore]: no actually in a PR i saw someone any how created the file in includes/ so thats why.

**Comment by championshuttler** [comment]
> It would be great if you could also remove the`.DS_Store` files as part of this patch.

Already did ;)

File: .gitignore (hunk 2 above)
Comment (muffinresearch) [reply .gitignore]: I add `.DS_Store` to master's `.gitignore` and then created a `.DS_Store` it's ignored. So I don't think you need the `**/.DS_Store` variant.

Comment (championshuttler) [reply .gitignore]: Thank you for investigation. Removed it :)

**Pull request merged**